### DISCUSSION TITLE
V11 Lighting API/Presets Support

### DISCUSSION THREAD

**Discussion by Jzuni97** [rnmapbox/maps#3159] [Q&A] [answered]
Hey all. Big thanks to the team for the new V11 support!
Regarding the lighting capabilities,  I see they have a new [API](https://github.com/mapbox/mapbox-gl-js/blob/main/MIGRATION_GUIDE_v3.md#lighting-api) for meddling with 3D layers. I was also curious about the lighting presets they have for the configuration style (see image). Is there currently methods or ways to interact with the configuration style and/or lighting preset?
<br>
<img width="1044" alt="Screenshot 2023-11-02 at 10 02 19 AM" src="https://github.com/rnmapbox/maps/assets/45020053/35b6f849-db73-490f-8f48-3c2ef3a9428b">

**Comment by mfazekas** [comment, marked as answer]
There is a V11 only StyleImport component to adjust the lightPreset and other configuration properties

https://rnmapbox.github.io/docs/examples/V11/StyleImportConfig

**Reply by Jzuni97** [reply]
Awesome, thanks for the help!
